lib/astutils.cpp

```diff
@@ -1263,18 +1264,21 @@ SmallVector<ReferenceToken> followAllReferences(const Token* tok,
         result.insert(refs.cbegin(), refs.cend());
 
         if (!inconclusive && result.size() != 1) {
+            SmallVector<ReferenceToken> refs_result;
             refs_result.push_back({tok, std::move(errors)});
             return refs_result;
         }
 
         if (!result.empty()) {
+            SmallVector<ReferenceToken> refs_result;
             refs_result.insert(refs_result.end(), result.cbegin(), result.cend());
```

Comment:
it's strange to use emplace_back, push_back, insert like this.. we should probably use the same method everywhere.

Reply:
`insert()` is for ranges. But that should probably use move iterators.

Reply:
ah yes.. missed it was a range. thanks..